src/openepd/model/specs/range/cmu.py

```diff
@@ -40,5 +41,5 @@ class CMURangeV1(BaseOpenEpdHierarchicalSpec):
     block_type: list[CmuBlockType] | None = pyd.Field(default=None, description="")
     insulated: bool | None = pyd.Field(default=None, description="")
     sound_performance: bool | None = pyd.Field(default=None, description="")
-    b1_recarbonation: AmountRangeGWP | None = pyd.Field(default=None, description="")
-    b1_recarbonation_z: RangeFloat | None = pyd.Field(default=None, description="")
+    b1_recarbonation: GwpKgCo2eStr | None = pyd.Field(default=None, description="")
```

Comment:
Isn't this a major change?

Reply:
The specs should be versioned differently. The idea was that we don't bump openEPD to major releases when we modify the specs, and if there are incompatible changes, we bump the major version of the spec and add the support. 

But, since again - no one is really using these properties (yet) it might be a good time to do a soft release w/out need for versioning exercise.